spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -494,6 +500,16 @@ public static boolean extensionsEnabled(SparkSession spark) {
     return extensions.contains("IcebergSparkSessionExtensions");
   }
 
+  public static boolean isPartitioned(SparkSession spark, Path tableLocation) {
+    try {
+      return Arrays.stream(
+              FileSystem.get(spark.sessionState().newHadoopConf()).listStatus(tableLocation))
```

Comment:
Should we use this `org.apache.hadoop.fs.FileSystem#get(java.net.URI, org.apache.hadoop.conf.Configuration)`

Reply:
yes, fixed. Thanks!